test/builder-api/src/origins.js

```diff
@@ -418,118 +404,82 @@ describe("Related Origin API functions", function () {
         });
     });
 
-    it("fails with a conflict when not deletable", function (done) {
-      request
-        .delete("/depot/origins/umbrella")
-        .set("Authorization", global.boboBearer)
-        .expect(409)
-        .end(function (err, res) {
-          expect(res.text).to.match(
-            /^There are 1 projects remaining in origin/
-          );
-          done(err);
-        });
-    });
-
-    it("succeeds in deleting associated project", function (done) {
-      request
-        .delete("/projects/umbrella/testapp")
-        .type("application/json")
-        .accept("application/json")
-        .set("Authorization", global.boboBearer)
-        .expect(204)
-        .end(function (err, res) {
-          expect(res.text).to.be.empty;
-          done(err);
-        });
-    });
-
-    it("succeeds", function (done) {
```

Comment:
you want to keep this test. Otherwise the origin is not deleted and that would explain your test failure in the build.